<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubDesignT1Class</class>
 <widget class="QMainWindow" name="SubDesignT1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SortComparison</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ShellSort">
       <property name="text">
        <string>ShellSort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="QuickSort">
       <property name="text">
        <string>QuickSort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HeapSort">
       <property name="text">
        <string>HeapSort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MergeSort">
       <property name="text">
        <string>MergeSort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>62</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="size">
       <property name="text">
        <string>样本量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time">
       <property name="text">
        <string>耗时：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="method">
       <property name="text">
        <string>排序方法：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>56</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="sizeNumber">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeNumber">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="methodText">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QChartView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="Generate"/>
    <addaction name="Open"/>
    <addaction name="Save"/>
    <addaction name="separator"/>
    <addaction name="Set"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Generate">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="Open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="Set">
   <property name="text">
    <string>参数</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ShellSort</tabstop>
  <tabstop>QuickSort</tabstop>
  <tabstop>HeapSort</tabstop>
  <tabstop>MergeSort</tabstop>
 </tabstops>
 <resources>
  <include location="SubDesignT1.qrc"/>
 </resources>
 <connections/>
</ui>
